nemo/collections/tts/models/magpietts_preference_optimization.py

```diff
@@ -26,18 +26,20 @@
 from nemo.collections.tts.models import MagpieTTSModel
 
 
-class MagpieTTSModelPrefDataGen(MagpieTTSModel):
-    """Small override to save inference metrics, used for datagen in Offline PO"""
+class MagpieTTSModelInference(MagpieTTSModel):
```

Comment:
Why change the name of the class? `MagpieTTSModelInference` is more indicative of what this class should be used for. It isn't a general purpose Inference version of the model, but rather a specific use case for alignment

Reply:
I changed it to `MagpieTTSModelPrefDataGen`